server/utils/viewUtils.ts

```diff
@@ -0,0 +1,7 @@
+import * as nunjucks from 'nunjucks'
+
+export const escape = (text: string): string => {
```

Comment:
Is this different from using the [safe flag](https://mozilla.github.io/nunjucks/templating.html#safe)?

Reply:
Yep, this is kind of the mirror of the safe flag. Before we can flag something as safe, we need to actually make it safe

My mental model is that in cases where you pass html into GDS Nunjucks components, that HTML is rendered with the safe flag, so that it doesn't get escaped, so we need to make sure we *do* escape anything we don't want rendered as raw HTML

Reply:
Thanks - that makes sense